app/javascript/components/test_iterations_summary_table.jsx

```diff
@@ -1,17 +1,17 @@
 import React, { useState, useEffect } from "react";
-import actionCable from "actioncable";
+import "actioncable";
```

Comment:
Do you need this LOC at all?

Reply:
Yes you do, otherwise the actioncable stuff is not loaded and does not work. I've checked

Reply:
Interesting. Thanks. I dislike that as magic!

Reply:
Hah. It's just a way for webpack to know what the component depends on.